### PR TITLE
RET-3773: Determine private hearing flag

### PR DESCRIPTION
**Pull request by JackReevies**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RET-3773

### Change description ###

Determine private hearing flag

**Does this PR introduce a breaking change?** (check one with "x")

```
[] Yes
[x] No
```

### CODE REVIEW

**Review by marcjudge — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/InitialConsiderationController.java [hunk 1]
```diff
@@ -40,6 +41,7 @@ public class InitialConsiderationController {
     private final InitialConsiderationService initialConsiderationService;
     private final DocumentManagementService documentManagementService;
     private final ReportDataService reportDataService;
+    private final CaseFlagsService caseFlagsService;
```
Comment (Itay-Kirsh) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/InitialConsiderationController.java:44]: give some space between private finals and private static finals. Maybe even move the latter above the former.

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/CaseFlagsService.java [hunk 2]
```diff
@@ -18,11 +23,15 @@
...
 import static uk.gov.hmcts.ecm.common.model.helper.CaseFlagConstants.VEXATIOUS_LITIGANT;
 import static uk.gov.hmcts.ecm.common.model.helper.Constants.NO;
+import static uk.gov.hmcts.ecm.common.model.helper.Constants.TSE_APP_RESTRICT_PUBLICITY;
 import static uk.gov.hmcts.ecm.common.model.helper.Constants.YES;
 
 @Slf4j
 @Service
 public class CaseFlagsService {
+
+    public static final String GRANTED = "Granted";
```
Comment (Itay-Kirsh) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/CaseFlagsService.java:33]: Needs to be public?

File: src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/CaseFlagsServiceTest.java [hunk 3]
```diff
@@ -4,29 +4,44 @@
...
 import static uk.gov.hmcts.ecm.common.model.helper.CaseFlagConstants.SIGN_LANGUAGE_INTERPRETER;
 import static uk.gov.hmcts.ecm.common.model.helper.CaseFlagConstants.VEXATIOUS_LITIGANT;
 import static uk.gov.hmcts.ecm.common.model.helper.Constants.NO;
+import static uk.gov.hmcts.ecm.common.model.helper.Constants.TSE_APP_CONSIDER_A_DECISION_AFRESH;
+import static uk.gov.hmcts.ecm.common.model.helper.Constants.TSE_APP_RESTRICT_PUBLICITY;
 import static uk.gov.hmcts.ecm.common.model.helper.Constants.YES;
 
 @ExtendWith(SpringExtension.class)
 class CaseFlagsServiceTest {
     public static final String CLAIMANT_NAME = "Claimant Name";
```
Comment (Itay-Kirsh) [inline src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/CaseFlagsServiceTest.java:41]: make all private

**Review by Itay-Kirsh — APPROVED** [review]

File: src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/CaseFlagsServiceTest.java (hunk 3 above)
Comment (Itay-Kirsh) [reply src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/CaseFlagsServiceTest.java:41]: this was meant for the public static finals

**Review by Itay-Kirsh — APPROVED** [review]

**Pull request merged**